e2e-tests/portal/cypress/e2e/journeys/payment-report-officer/utilisation-report-upload/can-upload-monthly-utilisation-report.spec.js

```diff
@@ -97,7 +97,80 @@ context('Monthly utilisation report upload', () => {
 
       utilisationReportUpload.checkReportTitle().should('exist');
       utilisationReportUpload.validationErrorTable().should('exist');
-      utilisationReportUpload.validationErrorTableRows().should('have.length.above', 0);
+      utilisationReportUpload.validationErrorTableRows().should('have.length', 9);
+
+      cy.assertValidationErrorTableRowContains({
+        tableRowIndex: 1,
+        message: 'Facility utilisation header is missing or spelt incorrectly',
+        exporter: '-',
+        row: '-',
+        column: '-',
+        entry: '',
+      });
+      cy.assertValidationErrorTableRowContains({
+        tableRowIndex: 2,
+        message: 'Fees paid to UKEF currency header is missing or spelt incorrectly',
+        exporter: '-',
+        row: '-',
+        column: '-',
+        entry: '',
```

Comment:
Could we update the `assertValidationErrorTableRowContains`  command to have some default params instead of having to pass empty"values here?

```js
const assertValidationErrorTableRowContains = ({
  tableRowIndex,
  message,
  exporter = '-',
  row = '-',
  column = '-',
  entry = '-',
}) => {
  // ...
};

module.exports = assertValidationErrorTableRowContains;
```

So then in tests like this we just need to pass custom/populated values and the command handles empty values 😎